File: modul 4/python/PRAK403-2410817110008-MuhammadAzmaAlFaqih.py
left, right = map(int, input().split())
Range = abs(left-right)

for i in range(0, Range+1):
    if left<right:
        print(left+i, right-i, end="",sep=" ")
    else:
        print(left-i, right+i, end="")

    if i<Range: print(" - ", end="")

File: modul 4/c/PRAK403_2410817110008_MuhammadAzmaAlFaqih.c
#include <stdio.h>
#include <stdlib.h>

int main(){
  int left, right;
  scanf("%d %d", &left, &right);
  int range = abs(left-right);
  for (int i = 0; i < range+1; i++){
    if (left < right){
      printf("%d %d", left+i, right-i);
    } else {
      printf("%d %d", left-i, right+i);
    }

    if (i < range){
      printf(" - ");
    }
  }
  printf("\n");
}
